Python/Product/PythonTools/PythonTools/Options/PythonDebuggingOptions.cs

```diff
@@ -139,13 +136,5 @@ public sealed class PythonDebuggingOptions {
             get;
             set;
         }
-
-        /// <summary>
-        /// True to use the legacy debugger. Default is false.
-        /// </summary>
-        public bool UseLegacyDebugger {
-            get;
-            set;
-        }
```

Comment:
We should delete `DebugProjectUITestsLegacyPtvsd` class now. That means the `UseVsCodeDebugger` there can be removed, and in the actual test code (DebuggerUITests/DebugProjectUITests) the `bool useVsCodeDebugger` parameter to each method can be removed. We can do that in another PR since that's a lot of changes.

FYI debug/attach to process still uses legacy ptvsd (it doesn't look at the setting), this is a feature we still need to implement for debugpy. So AttachUITests is to remain as-is until then.

Reply:
Yes, actual code removal is going to be large change. This one pretty much just hides legacy option. I think it may be better to do separate PR for the actual engine removal.